packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -567,6 +606,65 @@ QUnit.module('Chat', () => {
         });
     });
 
+    QUnit.module('Confirmation popup integration', moduleConfig, () => {
+        QUnit.test('passed rtlEnabled option value in Chat should be proxied to the Confirmation popup', function(assert) {
+            const items = [
+                { text: 'a', author: userFirst },
+                { text: 'b', author: userSecond },
+            ];
+
+            this.reinit({
+                user: userSecond,
+                editing: {
+                    allowDeleting: true
+                },
+                rtlEnabled: true,
+                items,
+            });
+
+            const $bubbles = this.getBubbles();
+            $bubbles.eq(1).trigger('dxcontextmenu');
+
+            const $deleteButton = this.getContextMenuItems().eq(0);
+            $deleteButton.trigger('dxclick');
+
+            const $popup = $(`.${CHAT_CONFIRMATION_POPUP_WRAPPER_CLASS}`);
+            assert.notStrictEqual($popup.find(`.${RTL_CLASS}`).length, 0, 'rtl class is passed to the popup');
+        });
+
+        QUnit.test('input should be focused after message delete popup is closed', async function(assert) {
+            const items = [
+                { text: 'a', author: userFirst },
+                { text: 'b', author: userSecond },
+            ];
+
+            this.reinit({
+                user: userSecond,
+                editing: {
+                    allowDeleting: true
+                },
+                items,
+            });
+
+            const $bubbles = this.getBubbles();
+            $bubbles.eq(1).trigger('dxcontextmenu');
+
+            const $deleteButton = this.getContextMenuItems().eq(0);
+            $deleteButton.trigger('dxclick');
+
+            const $popup = $(`.${CHAT_CONFIRMATION_POPUP_WRAPPER_CLASS}`);
+            const $cancelButton = $popup.find(`.${DX_BUTTON_CLASSNAME}`).last();
+
+            const done = assert.async();
+            $cancelButton.trigger('dxclick');
+
+            setTimeout(() => {
```

Comment:
Can we turn off animation and not make some tests asynchronious? Explore 'fx.off = true'